backend/dataall/modules/dataset_sharing/aws/kms_client.py

```diff
@@ -69,3 +69,22 @@ def check_key_exists(self, key_alias: str):
             return None
         else:
             return key_exist
+
```

Comment:
I like this method, but I don't see that it is being used at the moment. What feature are you trying to implement?

Reply:
We needed this for an internal implementation. I will remove it as its not being used anywhere in open source.